### PR TITLE
Jenkins integration

### PR DESCRIPTION
**Pull request by Loui4**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,6 +1,7 @@
...
 WORKDIR /app
+COPY package.json ./
+RUN npm install --only=prod
+RUN npm install pm2 -g
 COPY . .
-RUN cd /app && npm i --only=prod && cd ./client/ && npm i --only=prod
-RUN apk upgrade && apk add bash
-RUN npm i -g serve
\ No newline at end of file
+CMD [ "pm2-runtime", "server/server.j" ]
```
Comment (bwighane) [inline Dockerfile]: Does this work when  tested? I see you have missed an "s" at the end

**Review by bwighane — CHANGES_REQUESTED** [review]

**Comment by bwighane** [comment]
Hey @Loui4 , I simply forgot to submit after reviewing the commit. Apologies for that.

File: Dockerfile (hunk 1 above)
Comment (Loui4) [reply Dockerfile]: I tested it was working but I think I did some refactorings just before committing which I accidentally omitted the s.

Comment (bwighane) [reply Dockerfile]: Cool

**Review by bwighane — APPROVED** [review]

**Pull request merged**